.github/workflows/tests.yml

```diff
@@ -95,11 +95,11 @@ jobs:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
         python: ["3.8", "3.10", "3.12"]
-        node: [16.x, 18.x, 20.x]
-        include:  # `npm test` is running Windows find-visualstudio tests on an M1 Mac!!!
-          - os: macos-14
+        node: [18.x, 20.x, 22.x]
+        include:  # `npm test` runs Windows find-visualstudio tests on an Intel Mac!!!
```

Comment:
I think M1 Mac is more reasonable since intel not produced anymore

Reply:
Before we were mostly testing on Intel and only one test on M1.  This change reverses that.  Intel is still used broadly even if it is no longer being sold.

More important is why the Node.js v22 tests are failing.